blackbox/sqllogictest/src/sqllogictest.py

```diff
@@ -322,16 +312,18 @@ def main():
     parser = argparse.ArgumentParser(prog='sqllogictest.py', description=__doc__)
     parser.add_argument('-f', '--file',
                         type=str, required=True)
-    parser.add_argument('--hosts',
-                        type=str, default='http://localhost:4200')
+    parser.add_argument('--host',
+                        type=str, default='localhost')
+    parser.add_argument('--port',
+                        type=str, default='5433')
```

Comment:
Because locally I have a PostgreSQL running on 5432, so I used a value that would fit my dev environment :-)